YCloudYUSA/yusaopeny_activity_finder
# [Backport] Fix icon on the filters results

### How to review:
1. Go to Activity Finder 
2. Click the AGE button
3. Click ALL AGES(SKIP) button
4. Click ANY ACTIVITY(SKIP) button
5. Select the link  Anytime
6. Click the NEXT button
7. Click ALL LOCATION (SKIP)
![image](https://github.com/YCloudYUSA/yusaopeny_activity_finder/assets/22738130/b80ee0b0-dcd3-4fd9-885f-b3ec34280729)

Please ensure that there is no minus icon displayed when collapsing selected filters.
